lib/Main.js

```diff
@@ -17,6 +17,7 @@ var OAuth2 = require("./OAuth2");
 var Provisioning = require("./Provisioning");
 
```

Comment:
out of interest, what's the reason for having all the imports broken apart like this?